src/main/java/com/idrsolutions/microservice/JPedalServlet.java

```diff
@@ -236,16 +243,19 @@ protected boolean validateRequest(final HttpServletRequest request, final HttpSe
         return true;
     }
 
-    private static void convertPDF(final Mode mode, final String userPdfFilePath, final String outputDirStr,
-                                   final String fileNameWithoutExt, final Map<String, String> paramMap) throws Exception {
+    private static void convertPDF(final String uuid, final Mode mode, final String userPdfFilePath, final String outputDirStr,
+                                   final String fileNameWithoutExt, final Map<String, String> paramMap, final long maxDuration) throws Exception {
+        ErrorTracker durationTracker = new DurationTracker(uuid, maxDuration);
```

Comment:
Would I be correct in thinking this can be final?
```suggestion
        final ErrorTracker durationTracker = new DurationTracker(uuid, maxDuration);
```

Reply:
@daniel-warren while you are correct, I have change it so that the duration tracker is now created and passed to convertPDF. I think that is a better change